maintainers/google-season-of-docs-2024.md

```diff
@@ -0,0 +1,84 @@
+
+Measuring the project's success:
+
+Nix has a repository called nixpkgs which is our official package collection.
+It includes packages such as libraries and programs, but also nix helper for building software.
+Every package in this repo gets build by our ci and the build artifacts are uploaded to our binary cache (https://cache.nixos.org).
+If someone now was to cross compile something using nix tooling, nix would go and fetch the needed builder from the binary cache.
+We measure the number of cache hits per entry using our content delivery network provider.
+
+We would consider the cross compilation documentation successful if cache hits for said builder have a noticeable up tick in relation to overall nixpkgs usage.
+
+Measuring the success of docker build tools is a bit more tricky since everything nix needs to build docker images is basically gnu-coreutils and python with it's stdlib.
+This means a rise in cache hits for python and coreutils as result of better dockerTools documentation would basically be indistinguishable from for example a large company adopting nix as their primary build tool.
+What we can do instead is plot a chart of https://github.com/search?q=lang%3Anix+dockerTools&type=code and compare if the dockerTools usage goes up in relation to overall nixpkgs usage.
+
+We would consider the docker documentation successful if there is an exponential dockerTools usage increase compared to before and with relation to general nixpkgs usage.
+
+
+Timeline:
+
+| Dates            | Action Items                                       |
+| ---------------- | -------------------------------------------------- |
+| May              | Orientation / Gather existing documentation pieces |
+| June - September | Write documentation                                |
+| October          | Cleanup existing docs                              |
+| November         | Project completion                                 |
+
+
+## Budget
+
+| Budget item        | Amount | Running Total | Notes/justifications                                                                                |
+| ------------------ | ------ | ------------- | --------------------------------------------------------------------------------------------------- |
+| Technical Writer   | 8000   | 8000          | 2 People x 4000 each, one person for the docker task, and one person for the cross compilation task |
+| Volunteer stipends | 500    | 8500          | 1 Person, doing evaluation, gathering performance data, organizing/hiring technical writers         |
+| Total              |        | 8500          |                                                                                                     |
+
+
+## Additional information
+
+Previous experience with technical writers or documentation:
+
+@Janik-Haag did the technical writing one introduction course by Google and thus is aware of what to look for when reviewing documentation.
```

Comment:
```suggestion
@Janik-Haag did the technical writing for an introduction course by Google and thus is aware of what to look for when reviewing documentation.
```

Reply:
the course name is technical writing one introduction, I'll put a link to https://developers.google.com/tech-writing/one so it get's obvious.